src/OmniSharp.Abstractions/Models/v1/ChangeBufferRequest.cs

```diff
@@ -7,9 +9,13 @@ namespace OmniSharp.Models
     public class ChangeBufferRequest : IRequest
     {
         public string FileName { get; set; }
+        [JsonConverter(typeof(ZeroBasedIndexConverter))]
```

Comment:
JsonConverter is sealed, and there is no way, that I'm aware of to extend it so Newtonsoft will pick it up.  That was my first though too.